src/components/Device/Device.tsx

```diff
@@ -222,15 +222,60 @@ export const StyledAppleMacbookDevice = styled(BaseDevice)`
   }
 `
 
+const macbookSVG = () => (
```

Comment:
the devices are low-fidelity, updated with hi-fidelity here https://github.com/aretecode/konsilos/pull/11/commits/26a8ee14bd24b9d7389a9ceac212018e445247ce